modules/light-clients/08-wasm/types/client_state_test.go

```diff
@@ -92,6 +86,7 @@ func (suite *TypesTestSuite) TestStatusGrandpa() {
 	}
 }
 
+/*
```

Comment:
https://github.com/cosmos/ibc-go/pull/4851 required since we need to store a contract before calling any of the client state funcs and that involves calling mockVM's `StoreCode` and `Pin`.